src/c-api.adoc

```diff
@@ -485,6 +485,29 @@ RISC-V intrinsics examples:
 vint8m1_t __riscv_vadd_vv_i8m1(vint8m1_t vs2, vint8m1_t vs1, size_t vl); // vadd.vv vd, vs2, vs1
 ----
 
+=== CMO Intrinsics
+
+The RISC-V CMO extension provides instruction for perform operations on copies 
+of data in the memory hierarchy. 
+
+In order to access the RISC-V CMO intrinsics, it is necessary to
+include the header file `riscv_cmo.h`.
+
+The functions are only available if the compiler's `-march` string enables the
+required ISA extension.
+
+.CMO Extension Intrinsics
+[%autowidth]
+|===
+|*Prototype*                                                                              |*Instruction*        |*Extension*
+|`+void __riscv_cbo_clean(void *addr);+`                                                  |`cbo.clean`          |Zicbom
+|`+void __riscv_cbo_flush(void *addr);+`                                                  |`cbo.flush`          |Zicbom
+|`+void __riscv_cbo_inval(void *addr);+`                                                  |`cbo.inval`          |Zicbom
+|`+void __riscv_cbo_zero(void *addr);+`                                                   |`cbo.zero`           |Zicboz
+|`+void __riscv_cbo_prefetch(void *addr, const int rw, const int locality);+`             |`prefetch.[r][w]`    |Zicbop
```

Comment:
Is `__riscv_cbo_prefetch` a wrapper around `__builtin_prefetch` where there are 4 possible localities? RISC-V supports 5 localities, 4 NTL hint prefixes and 1 without any prefix.

Reply:
Yes, this is __builtin_prefetch's wrapper. The 4 NTL hint maping had been included in Prefetch Intrinsics part on this doc. So, this commit only support a wrapper for __builtin_prefetch on zicbop extension.

Reply:
Can you document how the different rw/locality map to different instruction sequences?

Reply:
> Can you document how the different rw/locality map to different instruction sequences?

Do you mean to expand to describe these mappings? 
Ok, I will send a new version tomorrow.